.gitignore

```diff
@@ -1,8 +1,14 @@
-.DS_Store
-/node_modules/
-/npm-debug.log
-/vendor/
+# Directories
+/.git
+/.github
+/.wordpress-org
```

Comment:
We still want to track it in this GitHub repo, right?

Reply:
Changed in [8b132b4](https://github.com/WordPress/two-factor/pull/683/commits/8b132b456562109eb78044e936b402f24c555b10).